waspc/cli/Command/Db/Migrate.hs

```diff
@@ -1,28 +1,90 @@
 module Command.Db.Migrate
-    ( migrate
+    ( migrateSave
+    , migrateUp
     ) where
 
+import System.IO.Error (IOError)
+import Control.Monad.Catch (catch)
 import Control.Monad.Except (throwError)
 import Control.Monad.IO.Class (liftIO)
+import qualified Path as P
+import Path.IO
```

Comment:
We are trying to avoid direct imports like this, can you make it qualified or list what you want to import?

Reply:
agreed, I forgot to qualify it, tnx for catching it.